libjob_queue/include/ert/job_queue/job_queue_manager.h

```diff
@@ -48,6 +48,8 @@ typedef struct job_queue_manager_struct job_queue_manager_type;
   bool job_queue_manager_job_failed( const job_queue_manager_type * manager , int job_index);
 
   job_status_type job_queue_manager_iget_job_status(const job_queue_manager_type * manager, int job_index);
+  time_t job_queue_imanager_get_status_timestamp(const job_queue_manager_type * queue);
```

Comment:
is the `i` in `imanager` intended or just a typo?

Reply:
Typo - thank you.

Reply:
Judging by the implementation in the .c file, this looks like a typo ;)